src/VisualStudio/Core/Def/Implementation/ChangeSignature/AddParameterDialog.xaml.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Windows;
+using System.Windows.Input;
+using Microsoft.VisualStudio.PlatformUI;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.LanguageServices.Implementation.IntellisenseControls;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Notification;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.ChangeSignature
+{
+    /// <summary>
+    /// Interaction logic for AddParameterDialog.xaml
+    /// </summary>
+    internal partial class AddParameterDialog : DialogWindow
+    {
+        public readonly AddParameterDialogViewModel ViewModel;
+        private readonly IntellisenseTextBoxViewModel _typeIntellisenseTextBoxView;
```

Comment:
```suggestion
        private readonly IntellisenseTextBoxViewModel _typeIntellisenseTextBoxViewModel;
```

Otherwise I keep thinking it's a IWpfTextView.